lib/app_profiler/backend/stackprof_backend.rb

```diff
@@ -3,13 +3,21 @@
 require "stackprof"
 
 module AppProfiler
-  module Profiler
-    DEFAULTS = {
-      mode: :cpu,
-      raw: true,
-    }.freeze
+  module Backend
+    class StackprofBackend < Base
+      NAME = :stackprof
```

Comment:
I think this should be a method. If we ever use Sorbet in this project, it would be easier to enforce subclasses implement this by exposing a method.

Reply:
Moved it to a class method